menus/_prompts.ts

```diff
@@ -88,48 +101,54 @@ export async function useExisting() {
 export async function getNetwork() {
   return (
     await prompt({
-      type: 'list',
-      name: 'network',
-      message: 'select network',
-      choices: [
-        {
-          name: 'peregrine testnet',
-          value: 'wss://peregrine.kilt.io/parachain-public-ws',
-        },
-        {
-          name: 'Spirit mainnet',
-          value: 'wss://spiritnet.api.onfinality.io/public-ws',
-        },
-      ],
+      prompt: {
+        type: 'list',
+        name: 'network',
+        message: 'select network',
+        choices: [
+          {
+            name: 'peregrine testnet',
+            value: 'wss://peregrine.kilt.io/parachain-public-ws',
+          },
+          {
+            name: 'Spirit mainnet',
+            value: 'wss://spiritnet.api.onfinality.io/public-ws',
+          },
+        ],
+      },
     })
   ).network
 }
 
 export async function getOrigin() {
```

Comment:
This should not be asked for the creation of a web3-login project. 

The origin is made from the Frontend port, which is ask to be typed right after this prompt.